media-fonts/fontawesome/metadata.xml

```diff
@@ -5,6 +5,10 @@
 		<email>hilobakho@gmail.com</email>
 		<name>Mykyta Holubakha</name>
 	</maintainer>
+	<maintainer type="person">
+		<email>bernardo@standard.ai</email>
+		<name>Bernardo Meurer</name>
+	</maintainer>
 	<maintainer type="project">
 		<email>proxy-maint@gentoo.org</email>
 		<name>Gentoo Proxy Maintainers Project</name>
```

Comment:
since it's just font files, can you add 
`<stabilize-allarches/>` here?

look at `app-emacs/rust-mode/metadata.xml` for example. right after maintainers is ok.

it'll help with stabilizations, all keywords can be stabilized at once, since no compiling involved and those are just files.

https://wiki.gentoo.org/wiki/Stable_request#Simultaneous_stabilization_on_all_architectures

Reply:
Oh, nice! Done!